patterns/about-details-description-with-bullets.php

```diff
@@ -0,0 +1,63 @@
+<?php
+/**
+ * Title: About Details Description With Bullets
+ * Slug: twentytwentyfive/about-details-description-with-bullets
+ * Categories: text, about, columns
+ * Description: About Details Description With Bullet.
+ *
+ * @package WordPress
+ * @subpackage Twenty_Twenty_Five
+ * @since Twenty Twenty-Five 1.0
+ */
+
+?>
+
+<!-- wp:group {"align":"full","style":{"spacing":{"padding":{"top":"var:preset|spacing|70","bottom":"var:preset|spacing|70","left":"var:preset|spacing|80","right":"var:preset|spacing|80"},"margin":{"top":"0","bottom":"0"}}},"layout":{"type":"constrained","contentSize":""}} -->
+<div class="wp-block-group alignfull" style="margin-top:0;margin-bottom:0;padding-top:var(--wp--preset--spacing--70);padding-right:var(--wp--preset--spacing--80);padding-bottom:var(--wp--preset--spacing--70);padding-left:var(--wp--preset--spacing--80)">
+	<!-- wp:columns {"align":"wide","style":{"spacing":{"blockGap":{"top":"var:preset|spacing|70","left":"var:preset|spacing|70"}}}} -->
+	<div class="wp-block-columns alignwide">
+		<!-- wp:column {"verticalAlignment":"top","width":"50%"} -->
+		<div class="wp-block-column is-vertically-aligned-top" style="flex-basis:50%">
+			<!-- wp:heading {"level":3,"fontSize":"x-large"} -->
+			<h3 class="wp-block-heading has-x-large-font-size">
+				<?php esc_html_e( 'Let’s hear about history and ancestry', 'twentytwentyfive' ); ?>
+			</h3>
+			<!-- /wp:heading -->
+		</div>
+		<!-- /wp:column -->
+
+		<!-- wp:column {"verticalAlignment":"center","width":"50%"} -->
+		<div class="wp-block-column is-vertically-aligned-center" style="flex-basis:50%">
+			<!-- wp:paragraph {"style":{"typography":{"textTransform":"uppercase"}},"textColor":"primary","fontSize":"small"} -->
```

Comment:
Since this describes the list below it, this should also be a heading. If the first is an H3, this needs to be an H4 😄